core/src/test/java/org/wso2/testgrid/core/command/CleanUpCommandTest.java

```diff
@@ -23,19 +29,54 @@
 public class CleanUpCommandTest extends PowerMockTestCase {
 
     private static final Logger logger = LoggerFactory.getLogger(CleanUpCommandTest.class);
+    private static final String infraParamsString = "{\"operating_system\":\"ubuntu_16.04\"}";
 
     @Mock
     private TestPlanUOW testPlanUOW;
 
+    private Product product;
+    private DeploymentPattern deploymentPattern;
+
+    private String actualTestPlanFileLocation;
+    private String workspaceDir;
 
     @Test
     public void testCleanup() throws Exception {
+        final String randomStr = StringUtil.generateRandomString(5);
         testPlanUOW = mock(TestPlanUOW.class);
         List<String> dataToDelete = new ArrayList<String>();
         dataToDelete.add("TP1");
         String grafanaUrl = "ec2-34-232-211-33.compute-1.amazonaws.com:3000";
 
         when(testPlanUOW.deleteDatasourcesByAge(10)).thenReturn(dataToDelete);
+        TestPlan testPlan = new TestPlan();
+        String productName = "wso2-" + randomStr;
+        actualTestPlanFileLocation = Paths.get("target", "testgrid-home", TestGridConstants.TESTGRID_JOB_DIR,
+                productName, TestGridConstants.PRODUCT_TEST_PLANS_DIR, "test-plan-01.yaml").toString();
+        workspaceDir = Paths.get(TestGridUtil.getTestGridHomePath(), TestGridConstants.TESTGRID_JOB_DIR,
+                productName).toString();
+
+
+        this.product = new Product();
+        product.setName(productName);
+
+        this.deploymentPattern = new DeploymentPattern();
+        deploymentPattern.setName("default-" + randomStr);
+        deploymentPattern.setProduct(product);
+
+
+        testPlan.setId("TP1");
+        testPlan.setTestRunNumber(1);
+        testPlan.setDeploymentPattern(deploymentPattern);
+        testPlan.setInfraParameters(infraParamsString);
+        testPlan.setDeployerType(TestPlan.DeployerType.SHELL);
+        testPlan.setScenarioTestsRepository(Paths.get(workspaceDir, "/workspace/scenarioTests").toString());
```

Comment:
Incorrect use of Paths API.

```suggestion
        testPlan.setScenarioTestsRepository(Paths.get(workspaceDir, "workspace", "scenarioTests").toString());
```